schemas/webhooktunnel-token-response.yml

```diff
@@ -0,0 +1,20 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      "Webhooktunnel Token Response"
+description: |
+    Token for connecting a worker to webhooktunnel proxy
+type:                       object
+properties:
+  id:
+    type: string
+    title: Proxy Connection ID
+    description: |
+      id for proxy connection
+  token:
+    type: string
+    title: Auth Token
+    description: |
```

Comment:
wouldn't decoding and jwt verification fail if it is not properly base64 encoded?